app/views/admin/comments/_comment.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="card">
+  <div class="row">
+    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 card-header">
+      <%= image_tag comment.user.attachment.avatar.url, height: 50, width: 50, class: "rounded pull-left" %>
```

Comment:
comment.user.attachment.avatar.url

should there not be some kind of helper or profile picture method in model for user?